recipes/seqan3/meta.yaml

```diff
@@ -0,0 +1,32 @@
+{% set name = "SeqAn3" %}
+{% set version = "3.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/seqan/{{ name|lower }}/releases/download/{{ version }}/seqan-3.0.0-with-submodules.tar.gz
+  sha256: ba8f0fd5d19289f95823235b69fdc5a12aba3578292463eecf750722e8662bd3
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - cmake
+
+test:
+  commands:
+    - test -e ${PREFIX}/include/seqan3/version.hpp
+    - test -e ${PREFIX}/include/seqan3/core/platform.hpp
+    - test -e ${PREFIX}/share/cmake/seqan3/seqan3-config.cmake
+
+about:
+  home: https://www.seqan.de
+  development: https://github.com/seqan/seqan3
+  license: BSD-3-Clause
+  license_family: BSD
+  license_file: LICENSE.md
+  summary: 'SeqAn3 is the new version of the popular SeqAn template library for the analysis of biological sequences.'
```

Comment:
Is it common to add
```
extra:
  recipe-maintainers:
    - ...
```
?

Reply:
I think that's only on conda-forge, as bioconda doesn't spawn off a repo for each recipe

Reply:
I did a grep, and sometime `recipe-maintainers:` is added, but not very often (about 400 out of 6400).
But updating the recipe will be part of our release procedure anyways.